__tests__/integration.test.js

```diff
@@ -380,6 +380,36 @@ describe('/api/articles', () => {
             expect(articles).toBeSorted('created_at', {ascending: true});
         })
     });
+    test('GET: 200 responds with the correct array of objects when passed a query', () => {
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then(({body}) => {
+            const {articles} = body;
+            expect(articles.length).toBe(1);
+            articles.forEach(article => {
+                expect(article.topic).toBe('cats')
+            })
+        })
+    });
+    test('GET: 200 sends an empty array when given a valid query with no matches', () => {
```

Comment:
This would be 404 as batman would be not found as we do not have a batman topic.

We would have a 200 with topic=paper as paper is a topic in the database but we have no article about it. 
Please, implement both variants 🙂